rasa_nlu/extractors/duckling_extractor.py

```diff
@@ -100,8 +101,8 @@ def persist(self, model_dir):
         # type: (Text) -> Dict[Text, Any]
         file_name = self.name+".json"
         full_name = os.path.join(model_dir, file_name)
-        with io.open(full_name, 'w') as f:
-            f.write(json.dumps({"dimensions": self.dimensions}))
+        with io.open(full_name, 'w', encoding="utf-8") as f:
```

Comment:
Instead of changing this, I'd rather change the `self.dimensions` array to only include unicode encoded strings. This would mean, we need to ensure that `available_dimensions` only returns unicode strings. Do you mind changing it in a way that the strings in the array that function creates are wrapped in `str(...)` and `from builtins import str` is imported?